backend/src/controllers/user.controller.ts

```diff
@@ -60,6 +68,17 @@ export const update = async (req: Request, res: Response): Promise<Response> =>
   }
   const update = req.body as Partial<IUser>;
   try {
+    if (update.email) {
+      const numberOfDocuments = await User.find({
```

Comment:
also `countDocuments()` is not required. We want to check if any record exists. When nothing found, numberOfUsers should be null